ts/WoltLabSuite/Core/Component/Image/Viewer.ts

```diff
@@ -0,0 +1,24 @@
+import { Fancybox } from "@fancyapps/ui";
+
+const LOCALS = ["cs", "de", "en", "es", "fr", "it", "ja", "lv", "pl", "sk"];
```

Comment:
typo (also in other lines): the correct spelling is locale / locales, not local / locals